### ISSUE TITLE
Small Contribution to One of Our Lessons Rules

### ISSUE THREAD

**Issue by rgaiacs** [swcarpentry/instructor-training#496]
I received [a issue](https://github.com/swcarpentry/lesson-example/issues/124) that mention the instructor training checkout and [a pull request](https://github.com/swcarpentry/lesson-example/pull/125) to fix the issue. The pull request need to be updated but in I didn't heard from the future instructor in 20 days (my reply was with less than 24 hours of the pull request submission). When is reasonable to close a pull request due inactivity and log it as a issue again tagged as good first time contribution.

**Comment by ErinBecker** [comment]
@rgaiacs Thanks for this question - I think it is a good one. However, I don't think I'm the one to decide the answer. I believe that is a question for the Maintainers Subcommittee to decide.

**Comment by rgaiacs** [comment]
> However, I don't think I'm the one to decide the answer. I believe that is a question for the Maintainers Subcommittee to decide.

I disagree. This is instructor training related and who is the Director of Instructor Training should mandate the policy for the issues/pull request that are part of instructor training checkout. I really don't care that we don't have a unified policy for non instructor training checkout related issues/pull request because I fully trust in our maintainers to make the best decision for the lessons and the project. For example, if they think that they should step in and finished a pull request because the release is next week and the pull request is need I'm happy with their decision and I'm also happy if they decided to wait six months because the issue is minor. But for instructor training checkout I really care that we have a unified policy because the issue/pull request will probably be the first real interaction of the future instructor with the broad community and we should make our best to ensure that this first interaction is amazing.

At http://lists.software-carpentry.org/pipermail/maintainers/2017-September/000377.html you mentioned https://docs.google.com/document/d/1NJrK6so3Vct4SyAZGlQWCi4dfAm2ICT1-oeyP6tcMgg/edit. The document doesn't mention any "responsability" of pull request authors.

**Comment by mkuzak** [comment]
I think it touches both, training and the maintainers. I would like if there is an agreement from trainers committee on how long the pull request could be hanging but also should be based on the maintainers' feedback. Especially that some content is more sensitive to time than other. In this case the broken template is not being fixed.

**Comment by ChristinaLK** [comment]
@rgaiacs I'll chime in and say that from the instructor training side, we mainly care that folks have made a pull request to understand how contribution works.  From the maintainer side, those pull requests don't need to be treated any differently than non instructor-training pull requests, although, as you observe, it would be great if we can ensure that the PR receives timely feedback as the instructor trainees are typically first-time contributors.

**Comment by ErinBecker** [comment]
Agreed with what @ChristinaLK and @mkuzak have added to the conversation. From the perspective of the maintainers group, it would be good to have a set of guidelines that the maintainers can use to guide their responses to contributions. Ideally this would include not only things like how long to leave a PR sitting before closing it but also how soon to respond to issues and PRs (to make sure that contributors' experiences are positive, as @rgaiacs and @ChristinaLK mentioned). Will this be part of the upcoming conversation with the Maintainers at the Community Call or should we initiate another conversation about this?

**Comment by ChristinaLK** [comment]
I'm not planning to talk about this specifically at the community call (although it may come up), but it should be something that the task force talks about.

**Comment by ErinBecker** [comment]
@rgaiacs @mkuzak - This issue is related to a number of broader conversations that the community is having regarding our maintenance process and contributor guidelines. @ChristinaLK and I have been talking individually with the SWC and DC lesson Maintainers and are working on formulating a comprehensive action plan to address the set of issues relevant to the Maintainer community. For now, I will tag this issue as "wait" to indicate that it is dependent upon other issues that are currently in discussion.
